README.md

```diff
@@ -64,21 +63,21 @@ Using GitLink via the command line is very simple:
 
 When working with a content proxy or an alternative git VCS system that supports direct HTTP access to specific file revisions use the `-u` parameter with the custom raw content root URL
 
-    GitLink.exe c:\source\catel -u https://raw.githubusercontent.com/catel/catel
+    GitLink.exe -u https://raw.githubusercontent.com/catel/catel <pdbfile>
```

Comment:
Isn't it weird to put the url in front? I would, from a tool perspective, expect this:

```
GitLink.exe <pdbfile> -u https://raw.githubusercontent.com/catel/catel
```

The reason for this is that we can slowly add more parameters based on the options the users wants to customize. But the real basic need is just `GitLink pdbfile'